### ISSUE TITLE
Extra element created after add a link on element with an attribute style.

### ISSUE THREAD

**Issue by philippeBchn** [froala/wysiwyg-editor#3026]
##### Expected behavior.
Add a link on a stylized word doesn't add an extra empty span.

##### Actual behavior.
Add a link on a word which has a style attribute will add an empty span after it, which has the same attribute style

##### Steps to reproduce the problem.
1) Write some words, add a break line (alt+enter)
```
At vero eos et accusamus 
et iusto odio dignissimos
```
2) Add some style (strong, color...)
```
<p>At vero eos et <strong><span style="color: rgb(139, 195, 74);">accusamus</span></strong>
<br>et iusto odio dignissimos</p>
```
3) Add a link on the last word of the first line

We expect to get this: 
```
<p>At vero eos et <strong><span style="color: rgb(139, 195, 74);"></span></strong><a href="https://www.froala.com/"><strong><span style="color: rgb(139, 195, 74);">accusamus</span></strong></a>
<br>et iusto odio dignissimos</p>
```

But I get that: 
```
<p>At vero eos et <strong><span style="color: rgb(139, 195, 74);"></span></strong><a href="https://www.froala.com/"><strong><span style="color: rgb(139, 195, 74);">accusamus</span></strong></a><strong><span style="color: rgb(139, 195, 74);"></span></strong>
<br>et iusto odio dignissimos</p>
```
After the link, there's an extra `<strong><span style="color: rgb(139, 195, 74);"></span></strong>`

I can't reproduice on the demo https://www.froala.com/wysiwyg-editor :(
I use the last version of Froala (2.8.4)

##### OS.
Mac OS High Sierra 10.13.6 

##### Browser.
Google Chrome 68.0.3440.106

**Comment by philippeBchn** [comment]
@stefanneculai of course, sorry to not did it before:

https://drive.google.com/file/d/1dVtyV4d5awfyoCyVJlxsFI15gU-ruFvR/view?usp=sharing

**Comment by stefanneculai** [comment]
This should be fixed in version 2.8.5. If you still face this problem, it would worth trying to replicate it in a jsFiddle as there might be happening just in specific configurations: https://jsfiddle.net/froala/wc5c3jhk/.
